compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/LargeMethodSplitter.java

```diff
@@ -295,8 +279,8 @@ private BLangFunction splitStopFunction(BLangPackage packageNode, SymbolEnv env)
         newFuncBody.stmts.add(stmts.get(stmts.size() - 1));
         generatedFunctions.add(newFunc);
 
-        // for the stop function, splitting is done same as the start function except here
-        // here just need only to call the next created stop function because the return value is () and not ()|error
+        // for the stop function, splitting is done same as the start function except here.
+        // here just need only to call the next created stop function because the return value is () and not ()|error.
```

Comment:
``` suggestion
        // For the stop function, splitting is done the same as the start function except here.
        // Here, it is only required to call the next created stop function because the return value is () and not ()|error.
```